src/rich-log/MessageBuilder.hh

```diff
@@ -41,7 +50,7 @@ public:
 
 public:
     template <class... Args>
-    MessageBuilder(rlog::location const& loc, Args&&... args) : _location(loc)
+    MessageBuilder(Args&&... args)
```

Comment:
Hm I don't think this is an improvement. Logging is instantiated many times and because we always know the first argument and can handle it immediately it should be faster this way (compile-time).